docs/en/sql-reference/10-sql-commands/50-administration-cmds/02-set-global.md

```diff
@@ -11,7 +11,7 @@ See also:
 ## Syntax
 
 ```sql
-SET [ GLOBAL ] <setting_name> = <new_value>
+SET [ SESSION | GLOBAL ] <setting_name> = <new_value>
 ```
 
 | Parameter | Description                                                                                                                                                                                          |
```

Comment:
The table below might need updates as well: 

SESSION / GLOBAL: Applies the setting change at the session or global level. If omitted, it applies to the session level by default. For more information about ....